2023-07-07 11:15:57.862 -03:00 [INF] Now listening on: https://localhost:7142
2023-07-07 11:15:57.884 -03:00 [INF] Now listening on: http://localhost:5203
2023-07-07 11:15:57.888 -03:00 [INF] Application started. Press Ctrl+C to shut down.
2023-07-07 11:15:57.889 -03:00 [INF] Hosting environment: Development
2023-07-07 11:15:57.890 -03:00 [INF] Content root path: D:\ProjetosEstudo\projects-maiu-and-blazor\BookStoreApp\BookStoreApp.API\
2023-07-07 11:16:02.094 -03:00 [INF] Made call to Weath Endpoint
2023-07-07 11:16:02.097 -03:00 [ERR] Fatal Error Ocurred.
System.Exception: This is our loging test exception
   at BookStoreApp.API.Controllers.WeatherForecastController.Get() in D:\ProjetosEstudo\projects-maiu-and-blazor\BookStoreApp\BookStoreApp.API\Controllers\WeatherForecastController.cs:line 27
2023-07-07 11:16:02.109 -03:00 [ERR] An unhandled exception has occurred while executing the request.
System.Exception: This is our loging test exception
   at BookStoreApp.API.Controllers.WeatherForecastController.Get() in D:\ProjetosEstudo\projects-maiu-and-blazor\BookStoreApp\BookStoreApp.API\Controllers\WeatherForecastController.cs:line 27
   at lambda_method2(Closure , Object , Object[] )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
